crates/sui-core/src/consensus_handler.rs

```diff
@@ -610,6 +610,16 @@ impl<C: CheckpointServiceNotify + Send + Sync> ConsensusHandler<C> {
         let leader_author = consensus_commit.leader_author_index();
         let commit_sub_dag_index = consensus_commit.commit_sub_dag_index();
 
+        let consensus_timestamp_bias = timestamp
+            - SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .unwrap()
+                .as_millis() as u64;
```

Comment:
```suggestion
        let consensus_timestamp_bias = timestamp.saturating_sub(
            SystemTime::now()
                .duration_since(UNIX_EPOCH)
                .unwrap()
                .as_millis() as u64);
```

Reply:
oh good point. let me also change it to i64